### PR TITLE
Update PA config folder location

### PR DESCRIPTION
**Pull request by sruti1312**
Signed-off-by: sruti1312 <srutiparthiban@gmail.com>

### Description
Updates the PA config folder location and add conf parameter when starting PA reader process. 

### Issues Resolved
https://github.com/opensearch-project/performance-analyzer/issues/188

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by sgup432 — APPROVED** [review]

**Comment by peterzhuamazon** [comment]
Hi @sruti1312 I am ok with this PR, waiting for review from the @opensearch-project/docs team.
Thanks.

**Review by Naarcha-AWS — APPROVED** [review]

**Review by alicejw-aws — APPROVED** [review]
LGTM

**Pull request merged**
